NPE if MIDI feature not present

To reproduce:
Run MidiKeyboard on a device that does not have the MIDI feature.
(Not sure how that can happen. Maybe it was side loaded by an OEM.)
Then Click one of the Program buttons.

EXPECT: no response

ACTUAL:
Caused by: java.lang.NullPointerException: 
  at com.mobileer.midikeyboard.MainActivity.midiSend (MainActivity.java:176)
  at com.mobileer.midikeyboard.MainActivity.midiCommand (MainActivity.java:157)
  at com.mobileer.midikeyboard.MainActivity.onProgramSend (MainActivity.java:111)

--- a/MidiKeyboard/src/main/java/com/mobileer/midikeyboard/MainActivity.java
+++ b/MidiKeyboard/src/main/java/com/mobileer/midikeyboard/MainActivity.java
@@ -170,14 +170,16 @@ public class MainActivity extends Activity {
     }
 
     private void midiSend(byte[] buffer, int count, long timestamp) {
-        try {
-            // send event immediately
-            MidiReceiver receiver = mKeyboardReceiverSelector.getReceiver();
-            if (receiver != null) {
-                receiver.send(buffer, 0, count, timestamp);
+        if (mKeyboardReceiverSelector != null) {
+            try {
+                // send event immediately
+                MidiReceiver receiver = mKeyboardReceiverSelector.getReceiver();
+                if (receiver != null) {
+                    receiver.send(buffer, 0, count, timestamp);
+                }
+            } catch (IOException e) {
+                Log.e(TAG, "mKeyboardReceiverSelector.send() failed " + e);
             }
-        } catch (IOException e) {
-            Log.e(TAG, "mKeyboardReceiverSelector.send() failed " + e);
         }
     }
 }

